### ISSUE TITLE
Add metalol to Championify as sources 

### ISSUE THREAD

**Issue by mrcut** [dustinblackman/Championify#292]
http://www.metalol.net/champions/lol-build-guide/solo-queue/Draven

**Comment by mrcut** [comment]
any news on it?

**Comment by dustinblackman** [comment]
I won't have time anytime soon to add it. Anyone is welcome to open a PR if they'd like to contribute.

**Comment by mrcut** [comment]
ok ty for your answer

2017-06-05 18:59 GMT+02:00 Dustin Blackman <notifications@github.com>:

**Comment by dustinblackman** [comment]
I've sent off an email to the authors of the site asking permission to add the source. If they agree, I'll include it in the next major release. Feel free to reach out to them and let know yourself too if you like.

**Comment by popcornAC** [comment]
@dustinblackman  Have they said yes or no? I've been looking for an open source project to contribute to, would love  to help.

**Comment by mrcut** [comment]
I didn't find a way to contact them there is not anything to contact them
on their website

2018-04-06 3:41 GMT+02:00 Arik C. <notifications@github.com>:

**Comment by mrcut** [comment]
Nevermind i just find out the mail , i ll tell you when i have an answer

2018-04-06 11:06 GMT+02:00 Mr Cut . <xxmrcutxx@gmail.com>:

**Comment by popcornAC** [comment]
Yeah I wasn't sure if Dustin had already received the OK from this as he added it to the milestone.

**Comment by dustinblackman** [comment]
They said yes, but I won't have anytime to work on this. PRs are welcome.

**Comment by PascaleBeier** [comment]
@dustinblackman can I tackle that?

**Comment by dustinblackman** [comment]
@PascaleBeier Feel free!
